tests/fixtures/issues/issue-000-invalid-versions.php

```diff
@@ -0,0 +1,604 @@
+<?php
+
+return [
```

Comment:
This is the invalid-versions fixture I've mentioned. Each entry is either from our logs, or taken from exisiting tests (and moved here). I tried to make sure each entry has a comment about why it's here, with a link to a source.